ui/apps/platform/src/Containers/NetworkGraph/simulation/SimulateNetworkPolicyButton.tsx

```diff
@@ -27,7 +27,7 @@ function SimulateNetworkPolicyButton({ simulation, isDisabled }: SimulateNetwork
             isDisabled={isDisabled || simulation.isOn}
             onClick={enableNetworkPolicySimulation}
         >
-            Simulate network policy
+            Network policy generator
```

Comment:
For your to-do lists, panel heading is **Network Policy Simulator** title case and different verbish noun.